README.rdoc

```diff
@@ -117,6 +117,29 @@ to build composite types by assigning an element encoder/decoder.
 PG::Coder objects can be used to set up a PG::TypeMap or alternatively
 to convert single values to/from their string representation.
 
+The following PostgreSQL column types are supported by ruby-pg (TE = Text Encoder, TD = Text Decoder, BE = Binary Encoder, BD = Binary Decoder):
+* Integer: {TE}[rdoc-ref:PG::TextEncoder::Integer], {TD}[rdoc-ref:PG::TextDecoder::Integer], {BE2}[rdoc-ref:PG::BinaryEncoder::Int2] {BE4}[rdoc-ref:PG::BinaryEncoder::Int4] {BE8}[rdoc-ref:PG::BinaryEncoder::Int8], {BD}[rdoc-ref:PG::BinaryDecoder::Integer]
```

Comment:
Maybe another bullet under integer here for the sized types:

```
* Integer: TE, TD, BD
  * BD: Int2, Int4, Int8
```

Do the classes themselves have documentation about when/why each should be used?

Reply:
Good idea! I'll change this.